packages/material-ui-icons/scripts/download.js

```diff
@@ -41,7 +42,7 @@ function downloadIcon(icon) {
       await fse.writeFile(
         path.join(
           __dirname,
-          `../material-io-tools-icons/ic_${icon.name}${themeFileNameMap[theme]}_24px.svg`,
+          `../material-icons/ic_${icon.name}${themeFileMap[theme]}_24px.svg`,
```

Comment:
We could lose the `ic_` prefix here and in the rename filter.

Reply:
done